schemas/chemotherapy.json

```diff
@@ -50,19 +50,31 @@
       }
     },
     {
-      "name": "chemotherapy_drug_name",
-      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "name": "drug_rxnormid",
```

Comment:
we can make this primaryId now right ?

Reply:
in rxnorm they call this rxcui not rxnormid i just thought it may be confusing for submitters

Reply:
changed to cui

Reply:
@blabadi We do not want to make this `primaryid` - that is reserved for the submitters primary keys.  this is actually and external reference and we want it to appear as `core` not `id` in the dictionary.